python-lib/config_handler.py

```diff
@@ -90,6 +90,25 @@ def add_scoring_config(dku_config, config):
         required=True,
         cast_to=NORMALIZATION_METHOD,
     )
+    dku_config.add_param(name="expert_mode", value=config.get("expert_mode", False), required=True)
+
+
+def add_timestamp_filtering(dku_config, config):
+    if dku_config.expert_mode:
+        based_threshold = dku_config["user_visit_threshold"]
+        dku_config.add_param(
+            name="top_n_most_recent",
+            value=config.get("top_n_most_recent"),
+            checks=[
+                {"type": "sup", "op": 0},
+                {
+                    "type": "sup_eq",
+                    "op": based_threshold,
+                    "err_msg": "The timestamp filtering value should be superior to the User visit threshold",
+                },
+            ],
```

Comment:
maybe we can remove this condition: it could make sense to only keep users with more than 15 visits but from those users keep only the 10 most recent visits (of items that were note filtered with visit_item_threshold)

Reply:
I wrote it as the two filters were generating no results (Because at first we were filtering to keep for example 20 results, so making the number visit equal to 25 was filtering all the results). But now that you are applying timestamp filtering after n visit filtering, it would indeed not have the same effect

Reply:
ok it makes sense. I'll remove this check now.

Reply:
b3a3442